routes/tools_routes.js

```diff
@@ -58,7 +58,8 @@ async function matchingCards(filter) {
       cards = Filter.filterCardsDetails(cards, filter);
     }
     if (Filter.filterUses(filter, 'price') || Filter.filterUses(filter, 'price_foil')) {
-      const withPrices = await addPrices(cards);
+      const withPricesPre = await addPrices(cards.map((card) => ({ details: card })));
```

Comment:
why are you doing this

Reply:
If we want to be able to use addPrices with cube card objects it needs to take them this way.